### PR TITLE
Adopt AIP-133: Create

### PR DESCRIPTION
**Pull request by rofrankel**
This PR has a few deliberate commits:

1.  Copy Google's AIP exactly.
2.  Rename .md to .md.j2 (probably could have combined this with (1) but oh well).
3.  Remove Google idioms and add AEP idioms.
4.  Isolate the protobuf-specific guidance and add an (empty) OAS tab.

This PR doesn't currently have OAS content.

### CODE REVIEW

**Comment by toumorokoshi** [comment]
nit: would it work for you if you forked and did a PR from your fork? It looks like you're pushing branches on the main repo.

I made the mistake of some pushes to the branches on this repo, and I worry we're going to clutter it up a bit.

File: aep/general/0133/aep.yaml
Comment (toumorokoshi) [inline aep/general/0133/aep.yaml:3]:
```suggestion
state: approved
```

File: aep/general/0133/aep.md.j2
Comment (toumorokoshi) [inline aep/general/0133/aep.md.j2:13]:
```suggestion
APIs **should** provide a create method for resources unless it is
```

File: aep/general/0133/aep.md.j2
Comment (toumorokoshi) [inline aep/general/0133/aep.md.j2:24]: nit: I've never liked the fact that we have to restate long-running everywhere: it's pretty redundant. I kind of feel like we should at most just reference long-running methods along the lines of "this may be a long running method", and consolidate guidance to LROs there.

thoughts?

File: aep/general/0133/aep.md.j2
Comment (toumorokoshi) [inline aep/general/0133/aep.md.j2:35]: strong consistency link is broken.

File: aep/general/0133/aep.md.j2
Comment (toumorokoshi) [inline aep/general/0133/aep.md.j2:55]: any reason this isn't a must?

File: aep/general/0133/aep.md.j2
Comment (toumorokoshi) [inline aep/general/0133/aep.md.j2:150]: we can remove this guidance - We were working toward removing AIP-128 from aip.dev anyway, in lieu of just adding guidance in the rest of the AIPs.

File: aep/general/0133/aep.md.j2
Comment (toumorokoshi) [inline aep/general/0133/aep.md.j2:119]: do we need this guidance? it's basically saying the same thing that AEP-151 says: if the request takes a long time, then use an LRO.

More duplicative guidance makes things harder to read, IMO.

**Review by toumorokoshi — CHANGES_REQUESTED** [review]
A couple small comments (a should that we should make a must, and a few sections to remove). otherwise LGTM!

File: aep/general/0133/aep.md.j2
Comment (rofrankel) [reply aep/general/0133/aep.md.j2:24]: Discussed live:  agreed on the approach, but we'll do a separate cleanup PR for this.  @toumorokoshi will file an issue.

File: aep/general/0133/aep.md.j2
Comment (rofrankel) [reply aep/general/0133/aep.md.j2:35]: I actually don't know what this is supposed to link to.  The link is broken at https://google.aip.dev/133 as well.

I'm just going to remove this bullet for now - we can always add it back later after we merge the planes AEP.

File: aep/general/0133/aep.md.j2
Comment (rofrankel) [reply aep/general/0133/aep.md.j2:55]: Nope, changed.

File: aep/general/0133/aep.md.j2
Comment (rofrankel) [reply aep/general/0133/aep.md.j2:24]: https://github.com/aep-dev/aep.dev/issues/106

File: aep/general/0133/aep.md.j2
Comment (rofrankel) [reply aep/general/0133/aep.md.j2:119]: Thanks for filing https://github.com/aep-dev/aep.dev/issues/106 - assuming we can potentially clean this up later and not block the PR on it.

File: aep/general/0133/aep.md.j2
Comment (rofrankel) [reply aep/general/0133/aep.md.j2:150]: Removed.

**Comment by rofrankel** [comment]
> nit: would it work for you if you forked and did a PR from your fork? It looks like you're pushing branches on the main repo.
> 
> I made the mistake of some pushes to the branches on this repo, and I worry we're going to clutter it up a bit.

Happy to switch to that workflow, though FWIW when a PR is merged the PR branch gets deleted.  Open (especially abandoned) PRs will still create clutter though.

File: aep/general/0133/aep.md.j2
Comment (toumorokoshi) [reply aep/general/0133/aep.md.j2:114]: I would have preferred this would be cleaned up in the future since you're touching the file already :P - but yes it's not a blocker.

File: aep/general/0133/aep.md.j2
Comment (toumorokoshi) [reply aep/general/0133/aep.md.j2]: I'd rather keep it actually - it's a good note and I don't want to forget re-adding it. It's also good guidance: eventual consistency is confusing to most users at first blush.

If the management plane reference is an issue, I'm fine with just making this a generic "SHOULD"

Here's the place it should link to: https://aep.dev/121#strong-consistency

**Review by toumorokoshi — CHANGES_REQUESTED** [review]
unresolved one comment - I think it's important to keep the clause around consistency. Otherwise LGTM.

File: aep/general/0133/aep.md.j2
Comment (rofrankel) [reply aep/general/0133/aep.md.j2]: Thanks for the actual link target. :) 

Regarding whether it's good guidance in general...I agree strong consistency is a nice thing to have, of course.  But if someone has a system architecture that doesn't allow for strong consistency, are they ever going to change that because of a **should** in the AEPs?  "Strong consistency is preferable" seems like A) a truism; B) more about system architecture than API design; and C) somewhat of a blinkered recommendation, as if the AEPs are oblivious to the system design challenges people might be facing when this guidance would apply.

WDYT?  If that's not persuasive, thoughts of filing an issue to revisit this?

Comment (toumorokoshi) [reply aep/general/0133/aep.md.j2]: I think there's plenty of things that are shoulds in the AEPs that one wouldn't change because it's in there - so I don't think that's a good reason to exclude this one. Similar to how an adopter of the AEP won't change their HTTP method to POST because the AEPs say so.

regardless of the difficulty of implementing it on the backend, good guidance is good guidance - and it's something people should consider, especially when authoring an API from scratch. The reason it's a *SHOULD* (I'd love to live in a world where it could be a *MUST*) is because the reality is some systems are hard to change, or is in direct conflict with backend behavior that is more desirable. 

Since this is already in the AIP, I'd rather see it merged in and file an issue to change it away from AIP guidance. To make this more of a principled decision: I'd agree with defaulting to AIP guidance and filing issues with the suggested change (thereby putting the onus on the person who disagrees with upstream guidance to argue for it).

Comment (rofrankel) [reply aep/general/0133/aep.md.j2]: Sure, leaving it in and filing an issue to bias towards not blocking PRs on controversial modifications.

**Review by toumorokoshi — APPROVED** [review]

**Pull request merged**
